content/en/docs/instrumentation/js/libraries.md

```diff
@@ -1,11 +1,48 @@
 ---
-title: Using instrumentation libraries
+title: Instrumenting libraries
```

Comment:
I don't like this title change. It implies the page is about how to instrument a library, but it leads with describing how to use instrumentation libraries and register instrumentations in your app.

I'd prefer to have two articles if we want to document how to write an instrumentation library.

Reply:
fixed